App/components/Checkbox/checkbox.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode, useState } from 'react'
+import Checked from '../../../assets/images/checked.svg'
+import {
+  CheckboxContainer,
+  CheckboxText,
+  UncheckedCB,
+  CheckedCB,
+} from './checkbox.styled'
+
+type Props = {
+  checked: boolean
+  children: ReactNode
+}
+
+const Checkbox = ({ checked, children }: Props) => {
```

Comment:
Should be exported here, without default at the end;
+Add export to the whole project's `index.ts`